functions/helloWorld/config.ts

```diff
@@ -11,7 +11,7 @@ export class HelloWorld extends Construct {
     new AleiosLambda(this, 'HelloWorldLambda', {
       functionName: 'hello-world',
       architecture: Architecture.ARM_64,
-      tracing: Tracing.PASS_THROUGH,
+      tracing: Tracing.ACTIVE,
```

Comment:
```suggestion
      tracing: Tracing.ACTIVE,
```
tracing: Tracing.PASS_THROUGH,